<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>importToDocBase</class>
 <widget class="QWidget" name="importToDocBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>302</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="dmHeaderText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Import Document Items</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Import information</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="textLabel3_2">
            <property name="text">
             <string>Select a &amp;File to import from:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>mFileRequester</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KUrlRequester" name="mFileRequester"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="_2">
            <item>
             <widget class="QLabel" name="textLabel1">
              <property name="text">
               <string>Import &amp;Schema:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>mSchemaCombo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mSchemaCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="mSchemaInfo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>66</height>
             </size>
            </property>
            <property name="text">
             <string>this is interesting information about the selected schema</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="_3">
            <item>
             <widget class="QLabel" name="textLabel3">
              <property name="text">
               <string>Insert all Items &amp;after</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>dmPositionCombo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="dmPositionCombo"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mTagGroup">
         <property name="title">
          <string>Tags</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kurlrequester.h</include>
  <include location="local">klineedit.h</include>
  <include location="local">kpushbutton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
